Mathlib/Topology/Sets/Closeds.lean

```diff
@@ -245,18 +245,19 @@ def Opens.complOrderIso : Opens α ≃o (Closeds α)ᵒᵈ where
 
 variable {α}
 
+@[simp, norm_cast] lemma Closeds.isAtom_coe [T1Space α] {s : Closeds α} :
+    IsAtom (s : Set α) ↔ IsAtom s :=
+  Closeds.gi.isAtom_iff' rfl
+    (fun t ht ↦ by obtain ⟨x, rfl⟩ := Set.isAtom_iff.1 ht; exact closure_singleton) s
+
 /-- in a `T1Space`, atoms of `TopologicalSpace.Closeds α` are precisely the
 `TopologicalSpace.Closeds.singleton`s.
 
 TODO: use `minimal_nonempty_closed_eq_singleton` to show that an atom in `TopologicalSpace.Closeds`
 in a T₀ space is a singleton. -/
```

Comment:
Is this TODO done?

Reply:
Nope. I'm just restating the existing stuff, not generalising it.

Reply:
I have now done it because it was easy